packages/replay-internal/src/replay.ts

```diff
@@ -907,51 +964,6 @@ export class ReplayContainer implements ReplayContainerInterface {
     }
   }
 
-  /**
-   * Handle when visibility of the page content changes. Opening a new tab will
-   * cause the state to change to hidden because of content of current page will
-   * be hidden. Likewise, moving a different window to cover the contents of the
-   * page will also trigger a change to a hidden state.
-   */
-  private _handleVisibilityChange: () => void = () => {
```

Comment:
same question as with syncpromise. But I feel like I'm missing something 😅

Reply:
here this is def. "needed" because we need to un-register the callbacks again, for which we need the same function instance. so we cannot use `fn.bind(this)` here. I added a comment for this here!